dpdd.tex

```diff
@@ -258,14 +259,11 @@ \subsection{Fluxes and Magnitudes}
 Because flux measurements on difference images (Prompt data products; \S~\ref{sec:level1}) are performed against a template
 and thus represent a flux difference, the measured flux of a source on the difference image can be negative. The flux can also go negative for faint sources in the presence of noise. Negative fluxes cannot be stored as (Pogson) magnitudes; $\log$ of a negative number is undefined. We therefore prefer to store fluxes, rather than magnitudes, in database tables\footnote{This is a good idea in general. E.g., given multi-epoch observations, one should always be averaging fluxes, rather than magnitudes.}.\dmreq{0347}
 
-We quote fluxes in units of ``maggie''. A maggie, as introduced by SDSS, is a linear measure of flux. It is defined so that an object having a flux of one maggie (integrated over the bandpass) has an AB magnitude of zero:
-\begin{equation}
-    m_{AB} = -2.5 \log_{10}(f/\mathrm{maggie})
-\end{equation}
+LSST fluxes are quoted in units of nanojansky\footnote{For details, please see LSST \citeds{Document-27758}.} (1 nJy = $10^{−35}$ Wm$^{−2}$Hz$^{−1}$). We will provide estimates of the relative and absolute photometric calibration errors, and the
+the normalized system response (for details see Section 3.3.4 in the LSST Science Requirements Document). 
 
-We chose to use maggies (as opposed to, say, Jansky) to allow the user to differentiate between two distinct sources of photometric calibration error: the error in relative (internal) calibration of the survey, and the error in absolute calibration that depends on the knowledge of absolute flux of photometric standards.
+We acknowledge that the large majority of users will want to work with magnitudes. For convenience, we plan to provide columns with (Pogson) magnitudes\footnote{These will most likely be implemented as ``virtual'' or ``computed'' columns.}, where values with negative flux will evaluate to \code{NULL}. Similarly, we will provide columns with flux expressed in Jy (and its error estimate that includes the relative and absolute calibration error contributions).
 
```

Comment:
Isn't it confusing to retain the "Jy" columns as a "Similarly" when the previous paragraph has just said that that is the primary delivery mechanism?  Or is this really saying that one column will be nJy and the other will be Jy?

Reply:
I agree. I propose instead: 

"LSST fluxes are quoted in units of nanojansky (1 nJy = 10−35 Wm−2Hz−1). We will provide columns with flux and flux errors as well as columns with estimates of the relative and absolute photometric calibration errors, and the normalized system response (for details see Section 3.3.4 in the LSST Science Requirements Document). 

We acknowledge that the large majority of users will want to work with magnitudes. For convenience, we plan in addition to provide columns with (Pogson) magnitudes and magnitude errors, where values with negative flux will evaluate to NULL."

Reply:
The rewrite is much better. One minor detail, I would strike "columns with" in the discussion of calibration errors. I polled the SST and the expectation is that the calibration error can be modeled as a small set of numbers without having to record per-object values.